dashboard/app/assets/stylesheets/application.scss

```diff
@@ -350,28 +350,74 @@ img.video_thumbnail {
     }
 
     .user_menu_arrow_down,
-    .user_menu_arrow_up {
+    .user_menu_arrow_up,
+    .create_menu_arrow_down,
+    .create_menu_arrow_up, {
       font-size: 30px;
     }
 
-    .user_options {
+    .user_options, {
       position: absolute;
       top: 55px;
       right: 55px;
       font-size: 14px;
       line-height: 20px;
     }
 
+    .create_options {
+      position: absolute;
+      top: 55px;
+      right: 55px;
+      font-size: 14px;
+      background-color: $white;
+      width: 200px;
+      a {
+        color: $teal;
+        font-weight: 900;
+        height: 70px;
+        width: 198px;
+        float: left;
+        border-left: 1px solid $charcoal;
+        border-right: 1px solid $charcoal;
+        border-top: 1px solid $charcoal;
+      }
+      a:hover {
+        color: $teal;
+        font-weight: 900;
```

Comment:
Does this work?  I thought our use of web fonts meant that we usually chose a different font face if we wanted a different weight.